server/src/metadata/migration-runner/migration-runner.service.ts

```diff
@@ -151,10 +153,16 @@ export class MigrationRunnerService {
             columnMigration,
           );
           break;
-        default:
-          throw new Error(
-            `Migration column action ${columnMigration.action} not supported`,
+        case 'relation':
```

Comment:
@Weiko For more readability, maybe you can create an enum for your actions so it's easier to read and understand what action are possible

Reply:
Good point, done 👍